### ISSUE TITLE
Dataset issues

### ISSUE THREAD

**Issue by chaoxiaoshui613** [PRIS-CV/Reserve_to_Adapt#1]
Is it reasonable for you to give code where the training set and the test set of the target domain share the same dataset?
